buildSrc/src/main/java/com/uber/okbuck/wrapper/BuckWrapperTask.java

```diff
@@ -52,6 +55,19 @@ void installWrapper() {
         }
     }
 
+    private static String toWatchmanIgnoredDirs(Set<String> ignoredDirs) {
+        if (ignoredDirs.isEmpty()) {
+            return "";
+        }
+
+        String ignore_exprs = ignoredDirs
```

Comment:
I was copying the pattern from `toWatchmanMatchers` below. Shall I update that as well? Not sure why snake_case was being used in the first place.